src/main/java/com/snowflake/kafka/connector/records/RecordService.java

```diff
@@ -566,7 +566,6 @@ public boolean shouldSkipNullValue(
       // get valueSchema
       Schema valueSchema = record.valueSchema();
       if (valueSchema instanceof SnowflakeJsonSchema) {
-        // TODO SNOW-916052: will not skip if record.value() == null
```

Comment:
valueschema will never be null because snowflake json converter always [returns a value schema
](https://github.com/snowflakedb/snowflake-kafka-connector/blob/master/src/main/java/com/snowflake/kafka/connector/records/SnowflakeJsonConverter.java#L31)